src/components/panel/panel.js

```diff
@@ -724,25 +749,33 @@ function MdPanelService($rootElement, $rootScope, $injector, $window) {
  * @returns {!MdPanelRef}
  */
 MdPanelService.prototype.create = function(config) {
-  var configSettings = config || {};
+  config = config || {};
 
+  // If the passed-in config contains an ID and the ID is within _trackedPanels,
+  // return the tracked panel.
+  if (angular.isDefined(config.id) && this._trackedPanels[config.id]) {
+    return this._trackedPanels[config.id];
+  }
+
+  // If no ID is set within the passed-in config, then create an arbitrary ID.
   this._config = {
+    id: config.id || 'panel_' + this._$injector.get('$mdUtil').nextUid(),
     scope: this._$rootScope.$new(true),
     attachTo: this._$rootElement
   };
-  angular.extend(this._config, this._defaultConfigOptions, configSettings);
+  angular.extend(this._config, this._defaultConfigOptions, config);
 
-  var instanceId = 'panel_' + this._$injector.get('$mdUtil').nextUid();
-  var instanceConfig = angular.extend({ id: instanceId }, this._config);
+  var panelRef = new MdPanelRef(this._config, this._$injector);
+  this._trackedPanels[config.id] = panelRef;
```

Comment:
`config.id` is undefined if no id is set!

Reply:
this._trackedPanels is set here but there is no chance of removing elements here. 
Example: Create config with id-property and create panel. So in the first run a new scope is set and the panelRef is cached in _trackedPanels. Now destroy() the panelRef and create a new panel-instance with the same id -> the scope is $destroy(ed) but the old panelRef with the destroyed scope is used.